code/modules/crafting/stack_recipes/recipes_bricks.dm

```diff
@@ -42,6 +42,9 @@
 /decl/stack_recipe/bricks/fire_source/kiln
 	result_type                 = /obj/structure/fire_source/kiln
 
+/decl/stack_recipe/bricks/fire_source/alembic
+	result_type                 = /obj/structure/fire_source/heater
```

Comment:
should this need a higher skill requirement than just the default? presumably it should be more difficult to make than a kiln, given it has an adjustable draught and all